hack/make/deploy/deploy.mk

```diff
@@ -3,6 +3,15 @@ DEBUG_LOGS ?= true
 PLATFORM ?= "kubernetes"
 HELM_CHART ?= config/helm/chart/default
 
+## Display the image name used to deploy the helm chart
+deploy/image-ref:
```

Comment:
I don't really know nice naming practices when it comes to make targets, could we make this more explicit?

Like:
- `deploy/print-image-ref`
- `deploy/echo-image-ref`
- `deploy/display-image-ref`
- `deploy/show-image-ref`
- `deploy/get-image-ref`